### DISCUSSION TITLE
Allow me to drag to sort the entries

### DISCUSSION THREAD

**Discussion by adelara** [Authenticator-Extension/Authenticator#688] [Q&A] [answered]
**Describe the feature you want:**
I want to sort/order the entries in Authenticator by way of dragging them up or down

**Why do you want this feature in Authenticator?**
Because most use two or three of the entries and want them to be in the top

**Comment by mymindstorm** [comment, marked as answer]
This is a feature already, you should be able to drag accounts in edit mode.

**Reply by RobMahn** [reply]
This does work, but it is cumbersome in a long list and difficult to see what is happening.  The elevator does not move as you are dragging, so you can only move a few places at a time.  I found to move further, I could hold down the right mouse button while rolling the mouse wheel or using the cursor up/down keys.

**Comment by adelara** [comment]
oh boy, I missed the pencil and tried to drag right away. Thanks !!
